core/src/main/java/com/mapzen/android/graphics/FeaturePickListener.java

```diff
@@ -7,7 +7,12 @@
  */
 public interface FeaturePickListener {
   /**
-   * Receive information about features found in a call to {@link #pickFeature(float, float)}.
+   * Receive information about features found in a call to
+   * {@link com.mapzen.tangram.MapController#pickFeature(float, float)}. Note that a feature refers
+   * to a {@link com.mapzen.android.graphics.model.Polygon} or
+   * {@link com.mapzen.android.graphics.model.Polyline} and is any
+   * {@link com.mapzen.tangram.MapData} drawn using non-point style. To receive pick information for
+   * points, use the {@link LabelPickListener} interface.
```

Comment:
"To receive pick information for points, including MapData drawn using points style, use the {@link LabelPickListener} interface."